assets/src/components/query/limit/index.js

```diff
@@ -28,6 +29,7 @@ export const QueryLimit = ( {
 			min={ min }
 			max={ max }
 			onChange={ onLimitChange }
+			{ ...rest }
```

Comment:
`RangeControl` can receive more props so by putting this here it allows us to pass those props through our wrapping component - I used this to pass through the `help` prop for instance.

In general, I think whenever we build a **component**  that wraps a third party component, we should make sure we pass through extra props not explicitly overridden in our custom component unless there's very good reason to not pass them through.  Among other reasons, this allows for us automatically picking up any new prop availability/behaviour in the wrapped component without having to refactor our custom component.